packages/devtools_app/integration_test/test_infra/run/run_test.dart

```diff
@@ -125,7 +125,7 @@ class TestRunner with IOMixin {
     bool updateGoldens = false,
     Map<String, Object> testAppArguments = const <String, Object>{},
   }) async {
-    Future<void> _run({required int attemptNumber}) async {
+    Future<void> run({required int attemptNumber}) async {
```

Comment:
Can we rename this `runTest` or something different than `run` so that it doesn't match the name of its encompassing method?